gnat2goto/driver/tree_walk.adb

```diff
@@ -1455,22 +1428,38 @@ package body Tree_Walk is
    ----------------------
 
    function Make_Assume_Expr (N : Node_Id; Assumption : Irep) return Irep is
+      Source_Location : constant Source_Ptr := Sloc (N);
+      Assume_Params : constant Irep := Make_Parameter_List;
+      Assume_Type : constant Irep := Make_Code_Type
+        (Parameters => Assume_Params,
+         Ellipsis => False,
+         Return_Type => CProver_Void_T,
+         Inlined => False,
+         Knr => False);
```

Comment:
On a related note, I've never seen anywhere where we set this to `True`. Could we maybe change the constructor to give this a default value?

Reply:
TBH we could probably just nuke it entirely and only emit the default value when generating JSON, but I'm not putting that into this PR